runners-msi/common/common-directories.wxi

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm OK with the changes then except for one thing...

We stopped distributing Runners a while back so if it has any files that are __not__ also in master, they could be out of date. But if it's all the same files, we should be OK.